src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImageInteractor.cpp

```diff
@@ -229,8 +232,6 @@ bool medVtkViewItkDataImageInteractor::SetViewInput(medAbstractData* data, int l
 
 void medVtkViewItkDataImageInteractor::initParameters(medAbstractImageData* data)
 {
-    d->imageData = data;
```

Comment:
To explain a little bit: this line is already done in medVtkViewItkDataImageInteractor::setInputData which call this initParameters function